### PR TITLE
[TW-444] Sentry integration

### PR DESCRIPTION
**Pull request by mark-anthony-postman**
Added Sentry integration page and updated left navigation

NOTE: We are working on a better screenshot that will show some actual Sentry issues

### CODE REVIEW

File: src/pages/docs/integrations/available-integrations/sentry.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+[Sentry](https://sentry.io/welcome/) is a real-time error tracking system for applications and APIs. It supports a wide range of frameworks and languages, including JavaScript, Node, Python, and more. Sentry provides a detailed view of handled and unhandled issues with a stack trace for easy debugging.
+
+To set up Sentry integration for you API, you'll need to create an [auth token](https://docs.sentry.io/api/auth/) in Sentry and configure your API in Postman. After you set up the integration, you can view Sentry issues in the Postman API builder.
+
+## Creating a Sentry auth token
+
+1. Sign in to [Sentry](https://sentry.io/auth/login/).
+1. Create a new [internal integration](https://docs.sentry.io/product/integrations/integration-platform/). Sentry automatically generates an auth token for the integration.
+1. Copy the new auth token.  (You might want to save the token somewhere safe.)
```
Comment (HeatherStenson) [inline src/pages/docs/integrations/available-integrations/sentry.md:24]:
```suggestion
1. Copy the new auth token. (You might want to save the token somewhere safe.)
```

File: src/pages/docs/integrations/available-integrations/sentry.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+## Configuring a Sentry integration
+
+1. Open your API by selecting **APIs** in the sidebar, and then selecting an API and a version.
+1. Select the **Monitoring** tab.
+1. Under **Connect Postman to your monitoring workflows**, select **Sentry**.
+1. Enter a **Nickname** to help you recognize the integration later.
+1. Enter your Sentry **Organization slug**. You can get the slug from your browser's address bar or in Sentry in **Settings > General Settings**. Use the format: `https://sentry.io/organizations/<organisation_slug>/`
```
Comment (HeatherStenson) [inline src/pages/docs/integrations/available-integrations/sentry.md:32]:
```suggestion
1. Enter your Sentry **Organization slug**. You can get the slug from your browser's address bar or in Sentry in **Settings > General Settings**. Use the format: `https://sentry.io/organizations/<organization_slug>/`
```

File: src/pages/docs/integrations/available-integrations/sentry.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+## Viewing Sentry issues
+
+After you set up a Sentry integration, information for Sentry issues is available in Postman. For each issue you can view the level, if it's handled or not, when it was first and last seen, the number of events, and the number of affected users.
+
+To view Sentry issues, open an API version and select the **Monitoring** tab. The most recent issues are listed under **Sentry**.
+
+* Expand an issue to see more details, or select an issue to open it in Sentry.
+* To view all Sentry issues, select **View all issues**.
+* Use the **Environment** dropdown list to filter issues by [Sentry environment](https://docs.sentry.io/product/sentry-basics/environments/).
```
Comment (HeatherStenson) [inline src/pages/docs/integrations/available-integrations/sentry.md:47]: To match the UI screenshot:

```suggestion
* Use the **Environments** dropdown list to filter issues by [Sentry environment](https://docs.sentry.io/product/sentry-basics/environments/).
```

**Review by HeatherStenson — APPROVED** [review]
Looks good! 🍊

**Review by jkonrath-postman — APPROVED** [review]
One little thing, but otherwise LGTM after you get the screenshot sorted.

**Comment by jkonrath-postman** [comment]
I'm guessing this isn't happening. I'll close the PR with no merge but leave the branch just in case.

**Pull request closed without merging**